src/app/unlocked-stage/unlocked-stage.component.html

```diff
@@ -1,20 +1,21 @@
 <div id="container" [hidden]="gameOver || success">
     <div id="scene">
-        <div id="human-health-text">{{language?.legend.human}}</div>
+      <div class="progress  progress-bar-vertical">
+        <div class="progress-bar progress-bar-danger" role="progressbar"  [ngStyle]="{'height': sprayValue + '%'}">
```

Comment:
can this be passed as variable  {'height': sprayValue + '%'} ?

Reply:
@shreyans29 this may not be useful to do so. We have one variable for sprayValue, and then another one which wraps it as a string would need to be created. Then passing the variable to style binding may not be a readable code.